### PR TITLE
Release notes for v6.0.8-1

### PR DESCRIPTION
**Pull request by alexmilowski**
Release notes for v6.0.8-1 (same as GitHub & confluence)

### CODE REVIEW

File: content/platforms/release-notes/k8s-6-0-8-1-2020-10.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-28 or redislabs/redis:6.0.8-28.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-1
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-1 or redislabs/services-manager:6.0.8-1 (Red Hat registry)
+
+## New features
+
+ * Modules can be configuration in the database custom resource.
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-0-8-1-2020-10.md:32]:
```suggestion
 * Redis [Modules](https://redislabs.com/redis-enterprise/modules/) can now be [configured](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_database_api.md#dbmodule) in the database custom resource.
```

File: content/platforms/release-notes/k8s-6-0-8-1-2020-10.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ * Support was added for **OpenShift 4.5**
+ * Support was added for **Kubernetes 1.18**
+ * Added support for the [Gesher admission control proxy](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/admission/GESHER.md) to provide an administrator the ability to setup delegation to avoid the need for administrator intervention on every namespaced deployed operator.
+
+## Important fixes
+
+ * Added the missing services rigger health check (RED47062)
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-0-8-1-2020-10.md:39]:
```suggestion
 * Added the missing Services Rigger health check (RED47062)
```

File: content/platforms/release-notes/k8s-6-0-8-1-2020-10.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-28 or redislabs/redis:6.0.8-28.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-1
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-1 or redislabs/services-manager:6.0.8-1 (Red Hat registry)
+
+## New features
+
+ * Modules can be configuration in the database custom resource.
+ * Support was added for **OpenShift 4.5**
```
Comment (amiramm) [inline content/platforms/release-notes/k8s-6-0-8-1-2020-10.md:33]: want to add links to OpenShift 4.5 and K8s 1.18 on their respective Websites? perhaps to their release notes?

**Review by amiramm — CHANGES_REQUESTED** [review]
Look good, thanks! Please see requested changes/comments

File: content/platforms/release-notes/k8s-6-0-8-1-2020-10.md (hunk 3 above)
Comment (alexmilowski) [reply content/platforms/release-notes/k8s-6-0-8-1-2020-10.md]: I added the links

**Review by amiramm — APPROVED** [review]

**Comment by amiramm** [comment]
@bmansheim Please merge. Thanks!

**Pull request merged**